lib/js/components/DatePickers/DatePickerBox/DatePickerBox.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+	<div
+		class="ds-datePickerBox"
+		:class="{
+			'-ds-disabled': state === DATE_PICKER_STATES.DISABLED,
+			'-ds-interactive': isInteractive,
+			'-ds-warning': color === DATE_PICKER_COLORS.WARNING,
+			'-ds-neutral': color === DATE_PICKER_COLORS.NEUTRAL,
+			'-ds-isOpen': isOpen,
+		}"
+	>
+		<span v-if="eyebrowText" class="ds-datePickerBox__eyebrow">{{ eyebrowText }}</span>
+		<div class="ds-datePickerBox__dateWrapper">
+			<ds-icon
+				v-if="icon"
+				class="ds-datePickerBox__icon"
+				:class="{ '-ds-hiddenOnMobile': isIconHiddenOnMobile }"
+				:icon="icon"
+				:size="ICON_SIZES.XX_SMALL"
+			></ds-icon>
+			<span class="ds-datePickerBox__date">{{ text }}</span>
+		</div>
+
+		<div v-if="state === DATE_PICKER_STATES.LOADING" class="ds-datePickerBox__loader">
+			<ds-icon :icon="ICONS.FAD_SPINNER_THIRD" :size="ICON_SIZES.SMALL" spinning />
+		</div>
+	</div>
+</template>
+
+<style scoped lang="scss">
+@import '../../../../styles/settings/typography/tokens';
+@import '../../../../styles/settings/colors/tokens';
+@import '../../../../styles/settings/spacings';
+@import '../../../../styles/settings/media-queries';
+@import '../../../../styles/settings/borders';
+@import '../../../../styles/settings/radiuses';
+
+@mixin color-scheme(
+	$color-eyebrow,
+	$color-date,
+	$color-icon,
+	$color-icon-hovered,
+	$color-date-hovered,
+	$color-icon-disabled,
+	$color-eyebrow-disabled,
+	$color-date-disabled,
+	$self
+) {
+	&.-ds-interactive {
+		&:hover {
+			#{$self}__icon {
+				color: $color-icon-hovered;
+			}
+
+			#{$self}__date {
+				color: $color-date-hovered;
+			}
+		}
+	}
+
+	&.-ds-disabled {
+		#{$self}__eyebrow {
+			color: $color-eyebrow-disabled;
+		}
+
+		#{$self}__date {
+			color: $color-date-disabled;
+		}
+
+		#{$self}__icon {
+			color: $color-icon-disabled;
+		}
+	}
+
+	&.-ds-isOpen {
+		&:hover {
+			#{$self}__icon {
+				color: $color-icon;
+			}
+
+			#{$self}__date {
+				color: $color-date;
+			}
+		}
+	}
+	#{$self}__eyebrow {
+		color: $color-eyebrow;
+	}
+
+	#{$self}__date {
+		color: $color-date;
+	}
+
+	#{$self}__icon {
+		color: $color-icon;
+	}
+
+	#{$self}__loader {
+		color: $color-icon;
+	}
+}
+
+.ds-datePickerBox {
+	$self: &;
+
+	border: $border-xs solid transparent;
+	display: flex;
+	flex-direction: column;
+	padding: $space-2xs $space-xs;
+	pointer-events: none;
+	position: relative;
```

Comment:
in figma we have min-width: 76px, should we add it here?

Reply:
https://www.figma.com/design/izQdYyiBR1GQgFkaOIfIJI?node-id=8505-9018#863153373